lib/fog/libvirt/models/compute/server.rb

```diff
@@ -363,7 +363,7 @@ def to_xml
                         end
                       end
 
-                      xml.target(:dev => target_device, :bus => args["bus_type"] == "virtio" ? "virtio" : "scsi")
+                      xml.target(:dev => target_device, :bus => ceph_args["bus_type"] == "virtio" ? "virtio" : "scsi")
```

Comment:
I merged this separate so it properly closed the issue. Doing 2 different things (fixing a bug and adding a new feature) in the same commit is something I prefer to avoid.

Reply:
Ok, I haven't seen the fix when I have started to work on it. I will rebase the merge on it.